sirepo/job_api.py

```diff
@@ -151,6 +151,29 @@ def api_runStatus():
     return _request()
 
 
+@api_perm.require_user
+def api_sbatchLogin():
+    r = _request_content(
+        PKDict(computeJobHash='unused', jobRunMode=sirepo.job.SBATCH),
+    )
+    r.sbatchCredentials = r.pkdel('data')
+    return _request(_request_content=r)
+
+
+@api_perm.require_user
+def api_simulationDb():
```

Comment:
I think we need a different name. This is about doing something that requires the simulation_db. It's similar to statelessCompute, but one level higher, so it should be a "compute" of some sort. statefulAnalysis, statefulCompute, statefulQuery, etc.

Reply:
l went with "statefulCompute". I think anaylsis should be left for things that need runDirs and the relationship between stateful and stateless is nice. daba63620